pkg/ccl/changefeedccl/show_changefeed_jobs_test.go

```diff
@@ -116,6 +119,39 @@ func TestShowChangefeedJobsBasic(t *testing.T) {
 	cdcTest(t, testFn, feedTestOmitSinks("webhook", "sinkless"), feedTestNoExternalConnection)
 }
 
+func TestShowChangefeedJobsShowsHighWaterTimestamp(t *testing.T) {
```

Comment:
nit: this test could use a short comment

Reply:
Added one, let me know what you think!

Reply:
Looks good!